docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  web-server:
+    build: .
+    ports:
+      - 4200:4200
+    volumes:
+      - .:/app
+    command: 
+      - bash
+      - -c
+      - |
+        if [ ! -f './node_modules/docker_init' ]; then
+          npm install --no-bin-links
```

Comment:
Couldn't the `--no-bin-links` be added directly in `Dockerfile`? So that you don't need to redefine the command here.

Reply:
In my head, docker-compose is for dev, swarm for large prod. Dockerfile is used by both or for small prod.
`--no-bin-links` option increase the size of image by a lot, so I would avoid making a docker file creating a large image for prod. I decide (but I can be changed no pb) that for dev this is acceptable. So docker-compose erase docker file command